media-libs/openimageio/patches/openimageio-1.8.10.patchset

```diff
@@ -83,5 +83,37 @@ index dacdc2e..4918389 100644
    ifeq (${uname},cygwin)
      platform := windows
 -- 
-2.15.0
+2.16.2
+
+
+From 89846363da62f9addd782efafaeb9d9beae65fee Mon Sep 17 00:00:00 2001
+From: =?UTF-8?q?Zolt=C3=A1n=20Mizsei?= <zmizsei@extrowerk.com>
+Date: Sun, 15 Apr 2018 11:28:02 +0200
+Subject: Build fix
+
+
+diff --git a/src/libutil/strutil.cpp b/src/libutil/strutil.cpp
+index c26e210..c9dfbf9 100644
+--- a/src/libutil/strutil.cpp
++++ b/src/libutil/strutil.cpp
+@@ -1140,7 +1140,7 @@ Strutil::strtof (const char *nptr, char **endptr)
+     std::locale native; // default ctr gets current global locale
+     char nativepoint = std::use_facet<std::numpunct<char>>(native).decimal_point();
+     // If the native locale uses decimal, just directly use strtof.
+-    if (nativepoint = '.')
++    if (nativepoint == '.')
```

Comment:
This is an upstream bug that will effect everyone, could you please open an PR there with this patch?

Reply:
https://github.com/OpenImageIO/oiio/pull/1918